index.html

```diff
@@ -1,19 +1,133 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+<title>Karma</title>
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+<link rel="stylesheet" href="css/normalize.css">
+<link rel="stylesheet" href="css/navBar.css">
+<link rel="stylesheet" href="css/style.css">
+<link rel="stylesheet" href="css/footer.css">
+<link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<body>
+    <!-- HEADER -->
+    <header class="header">
+        <div>
+            <img class="header-image" src="img/karma-logo.svg" alt="karma logo"/>
+        </div> 
+
+<!-- Add your HTML markup here -->
+<!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+<!-- All the images you need are in the 'img' folder -->
+        <!-- HEADER NAVIGATION -->
+        <nav>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <a class="nav-link" href="./index.html"><strong>Meet Karma</strong></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">How it Works</a> 
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="./registration.html">Store</a> 
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Blog</a> 
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Help</a> 
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Login</a> 
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+</body>
+    <!-- MAIN -->
+    <main>
+        <!-- TOP HALF OF PAGE -->
+        <div class="page-top-half">
```

Comment:
`page-top-half` is a intuitive name, however, it's not quite right, for the same reasons `section-1` is not -- basically, you cannot guarantee this section will remain on the top half. Same for bottom, etc.